src/backends/pgfplotsx.jl

```diff
@@ -1093,6 +1096,37 @@ function pgfx_axis!(opt::PGFPlotsX.Options, sp::Subplot, letter)
                 axis[:gridstyle],
             ),
         )
+
+        # minor ticks
+        # NOTE: PGFPlots would provide "minor x ticks num", but this only places minor ticks
+        #       between major ticks and not outside first and last tick to the axis limits.
+        #       Hence, we hack around with extra ticks. Unfortunately this conflicts with
+        #       `:zerolines` framestyle hack. So minor ticks are not working with
+        #       `:zerolines`.
+        minor_ticks = get_minor_ticks(sp, axis, ticks)
+        if minor_ticks !== nothing
+            minor_ticks =
+                ispolar(sp) && letter == :x ? [rad2deg.(minor_ticks)[3:end]..., 360, 405] :
```

Comment:
I copied this from above. Why do we skip the second tick here as well?

Reply:
that code was written by @pkofod and I just translated it. The best way to find out is to test what happens if you include it ^^;

Reply:
OK, no that does not work how I expected. I'll leave it as it is.

Reply:
:shrug: I don't remember much about this :)